pom.xml

```diff
@@ -7,7 +7,25 @@
     <groupId>org.example</groupId>
     <artifactId>MixCinema</artifactId>
     <version>1.0-SNAPSHOT</version>
-    <packaging>war</packaging>
+    <dependencies>
+        <dependency>
+            <groupId>org.hibernate</groupId>
+            <artifactId>hibernate-core</artifactId>
+            <version>5.4.21.Final</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.hibernate</groupId>
+            <artifactId>hibernate-java8</artifactId>
+            <version>5.4.21.Final</version>
+        </dependency>
+
+        <dependency>
+            <groupId>mysql</groupId>
+            <artifactId>mysql-connector-java</artifactId>
+            <version>8.0.21</version>
+        </dependency>
+    </dependencies>
```

Comment:
do you need war plugin here?
also, you might not need `maven-compiler-plugin` because it is already present in `<pluginManagement>` section

Reply:
done